.github/workflows/tests.yml

```diff
@@ -72,6 +72,7 @@ jobs:
     strategy:
       matrix:
         target: [
+          x86_64-unknown-linux-gnux32,
```

Comment:
IIUC this will test the Linux backend, not the fixed `rdrand` one.

Reply:
It will also test the rdrand backend, all x86 targets test it when you run `cargo test`

Reply:
Ah, true.